### PR TITLE
[4.0] small screen toolbar toggle fix

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #23815 .

### Summary of Changes
Changes in ``` administrator/templates/atum/index.php```

### Testing Instructions
Open the articles and reduce the browser viewport size

### Expected result
Toolbar displayed/hidden when toggle button is clicked

### Actual result
The toolbar toggle button does nothing

### Documentation Changes Required
None

### CODE REVIEW

**Comment by hardik-codes** [comment]
@ciar4n please have a look and suggest me changes if any.

File: administrator/templates/atum/index.php [hunk 1]
```diff
@@ -104,14 +104,16 @@
 				<?php // Subheader ?>
 				<a class="btn btn-subhead d-md-none d-lg-none d-xl-none" data-toggle="collapse" data-target=".subhead-collapse"><?php echo Text::_('TPL_ATUM_TOOLBAR'); ?>
 					<span class="icon-wrench"></span></a>
-				<div id="subhead" class="subhead">
-						<div id="container-collapse" class="container-collapse"></div>
-						<div class="row">
-							<div class="col-md-12">
-								<jdoc:include type="modules" name="toolbar" style="no" />
+					<div class="subhead-collapse" id="atumJsData" data-tmpl-sticky="<?php echo $stickyBar; ?>" data-tmpl-offset="<?php echo $offset; ?>">
```
Comment (ciar4n) [inline administrator/templates/atum/index.php]: What is the reason for setting these attributes (data-tmpl-sticky & data-tmpl-offset)? Where are these variables defined?

Comment (ciar4n) [reply administrator/templates/atum/index.php]: Remove the `data-tmpl-sticky` & `data-tmpl-offset` attributes and this should be fine.

Comment (hardik-codes) [reply administrator/templates/atum/index.php]: > Remove the `data-tmpl-sticky` & `data-tmpl-offset` attributes and this should be fine.

Will do it right away.

File: administrator/templates/atum/index.php [hunk 2]
```diff
@@ -104,14 +104,16 @@
...
+							<div id="container-collapse" class="container-collapse"></div>
+							<div class="row">
+								<div class="col-md-12">
+									<jdoc:include type="modules" name="toolbar" style="no" />
+								</div>
 							</div>
-					</div>
-				</div>
+						</div>
+					</div>
```
Comment (Quy) [inline administrator/templates/atum/index.php]:
```suggestion
					</div>
```

File: administrator/templates/atum/index.php [hunk 3]
```diff
@@ -104,14 +104,16 @@
 				<?php // Subheader ?>
 				<a class="btn btn-subhead d-md-none d-lg-none d-xl-none" data-toggle="collapse" data-target=".subhead-collapse"><?php echo Text::_('TPL_ATUM_TOOLBAR'); ?>
 					<span class="icon-wrench"></span></a>
-				<div id="subhead" class="subhead">
-						<div id="container-collapse" class="container-collapse"></div>
-						<div class="row">
-							<div class="col-md-12">
-								<jdoc:include type="modules" name="toolbar" style="no" />
+					<div class="subhead-collapse" id="atumJsData">
+				        <div id="subhead" class="subhead">
```
Comment (Quy) [inline administrator/templates/atum/index.php]:
```suggestion
						<div id="subhead" class="subhead">
```

**Comment by hardik-codes** [comment]
@ciar4n please review the changes

File: administrator/templates/atum/index.php [hunk 4]
```diff
@@ -104,14 +104,16 @@
 				<?php // Subheader ?>
 				<a class="btn btn-subhead d-md-none d-lg-none d-xl-none" data-toggle="collapse" data-target=".subhead-collapse"><?php echo Text::_('TPL_ATUM_TOOLBAR'); ?>
 					<span class="icon-wrench"></span></a>
-				<div id="subhead" class="subhead">
-						<div id="container-collapse" class="container-collapse"></div>
-						<div class="row">
-							<div class="col-md-12">
-								<jdoc:include type="modules" name="toolbar" style="no" />
+					<div class="subhead-collapse" id="atumJsData">
```
Comment (C-Lodder) [inline administrator/templates/atum/index.php]: What's the `atumJsData` ID for?

Comment (hardik-codes) [reply administrator/templates/atum/index.php]: Actually I gave this ID in analogy to j3.9

**Comment by hardik-codes** [comment]
Members please test this PR

File: administrator/templates/atum/index.php (hunk 4 above)
Comment (C-Lodder) [reply administrator/templates/atum/index.php]: @hardik-codes I don't believe it's being used in J4 though

Comment (hardik-codes) [reply administrator/templates/atum/index.php]: Ok then I'll be removing the ID.

**Comment by softforge** [comment]
Sorry, I am not sure what we should see. Are you talking about the Toggle menu button when in 
/administrator/index.php?option=com_fields&context=com_content.article
The toggle seems to work before the patch is applied even when the page is made small
Could you show an image of it in the broken and patched state so I can work out if I am looking at the right thing.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24083">issues.joomla.org/tracker/joomla-cms/24083</a>.</sub>

**Comment by hardik-codes** [comment]
@flyingwombats 
 Resize your browser window and you will find a toolbar button
tp-1

Earlier this button was not toggleable but after the patch it is toggleable.

**Comment by Bodge-IT** [comment]
Tested this and the toggle now works but no styling and no cursor change on hover, were these excpected?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24083">issues.joomla.org/tracker/joomla-cms/24083</a>.</sub>

**Comment by hardik-codes** [comment]
@Bodge-IT 
I think  the only intention was to make the toolbar button toggleable.

**Comment by hardik-codes** [comment]
@Bodge-IT 
I think  the only intention was to make the toolbar button toggleable.

**Comment by Bodge-IT** [comment]
I have tested this item :white_check_mark: successfully on 0ebeb6bd80bc7764aac9559cc770d2f8ac5fcf79<br /><br />Toggle button toggles after patch.

Note: No cursor indication of "button"<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24083">issues.joomla.org/tracker/joomla-cms/24083</a>.</sub>

**Comment by brianteeman** [comment]
Note this fails almost every measure of accessibility

**Comment by Quy** [comment]
This is no longer an issue with the new backend template. Please test PR #25570.

toolbar


<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24083">issues.joomla.org/tracker/joomla-cms/24083</a>.</sub>

**Pull request closed without merging**